xiblint/__main__.py

```diff
@@ -31,7 +32,11 @@ def main():
     )
     parser.add_argument("-v", "--version", action="version",
                         version=__version__)
-    parser.parse_args()
+    parser.add_argument("--reporter", choices=("raw", "json"),
+                        default="raw",
+                        help="custom reporter to use (optional)")
```

Comment:
argparse arguments are always optional, so perhaps we can omit the "(optional)". The default value "raw" would be indicated as part of the help text anyway.